regression/pages/lms/course_page_lms.py

```diff
@@ -22,3 +22,13 @@ def click_resume_button(self):
         Clicks Resume button of the course selected
         """
         self.q(css='.last-accessed-link').first.click()
+
+    def get_pages_name(self):
```

Comment:
It will output something like this [u'Home\n, current location', u'Course', u'Discussion', u'Wiki', u'Progress', u'Instructor']. It will return pages in sequence. I am sorry I don't have any clue about putting it before. Its in LMS. Can you pin point?